src/pvecontrol/config.py

```diff
@@ -12,16 +12,24 @@
             "password": confuse.Optional(str, None),
             "token_name": confuse.Optional(str, None),
             "token_value": confuse.Optional(str, None),
+            "timeout": confuse.Optional(int, default=0),
             "node": confuse.Optional(
                 {
                     "cpufactor": confuse.Optional(float, None),
                     "memoryminimum": confuse.Optional(int, None),
                 },
                 default={},
             ),
+            "vm": confuse.Optional(
+                {
+                    "max_last_backup": int,
```

Comment:
```suggestion
                    "max_last_backup": confuse.Optional(int, default=1500)
```

linked to https://github.com/enix/pvecontrol/pull/28/files#r1967529869

Reply:
I had to scratch my head a bit, but there is actually a default value set already in `config_default.yaml`, which is `1500`. Then, if `clusters.vm.max_last_backup` isn't set, it is defaulted to `vm.max_last_backup`.